assets/src/design-system/components/dropDown/menu/components.js

```diff
@@ -49,15 +56,98 @@ MenuContainer.propTypes = {
 };
 
 export const ListGroup = styled.ul`
+  box-sizing: border-box;
   list-style-type: none;
-  padding: 0;
+  margin: 6px 0;
+  display: block;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  width: 100%;
 `;
 
-export const ListItemLabel = styled.li``;
-
-export const ListItem = styled.li`
-  width: 100%;
+export const ListItemLabel = styled.li`
+  display: flex;
+  padding: 6px 2px 6px 8px;
+  margin: 4px 8px;
+  align-items: center;
 `;
 
-export const NoOptionsContainer = styled.div``;
-export const NoOptionsMessage = styled(Text)``;
+export const ListItem = styled.li(
+  ({ disabled, isSelected, theme }) => css`
+    display: grid;
+    grid-template-columns: 16px 1fr;
+    padding: 6px 2px 6px 8px;
+    margin: 4px 8px;
+    border-radius: ${theme.borders.radius.small};
+    align-items: center;
+    background-color: ${isSelected
+      ? theme.colors.interactiveBg.brandNormal
+      : 'inherit'};
+    cursor: ${disabled ? 'default' : 'pointer'};
+
+    ${themeHelpers.focusableOutlineCSS(theme.colors.border.focus)};
+
+    &:focus {
+      outline: none;
+      background-color: ${isSelected
+        ? theme.colors.interactiveBg.brandNormal
+        : theme.colors.interactiveBg.brandHover};
+    }
+
+    &:hover {
+      background-color: ${isSelected
+        ? theme.colors.interactiveBg.brandNormal
+        : theme.colors.interactiveBg.brandHover};
+    }
+
+    &:active {
+      background-color: ${isSelected
+        ? theme.colors.interactiveBg.brandActive
+        : theme.colors.interactiveBg.brandPress};
+    }
```

Comment:
We don't have any default hover, press, active colors for this component. For some interactivity and consistency, I’ve chosen to use the `brand` interactiveBg colors instead of just the `interactiveBg.active` so that when hover/active occur it's less abrupt.  

I've documented this in my google doc of questions here: https://docs.google.com/document/d/1w06W9T3sxbiqveoz9Zk9_sO87JPO4seQzjo1FIDBK54/edit#